app/i18n/nl/conf.php

```diff
@@ -178,7 +178,7 @@
 			'view' => 'Weergave',
 		),
 		'hide_read_feeds' => 'Categorieën en feeds zonder ongelezen artikelen verbergen (werkt niet met “Toon alle artikelen” configuratie)',
-		'img_with_lazyload' => 'Gebruik "lazy load" methode om afbeeldingen te laden',
+		'img_with_lazyload' => 'Gebruik <em>lazy load</em> methode om afbeeldingen te laden',
```

Comment:
why here `<em>` and in the other languages not?

Reply:
@Frenzie's personal preference for Dutch. We could update the other languages as well, but we can also leave it up to the translators

Reply:
Done